### PR TITLE
ci: use old red-arrow until the packages of Apache Arrow 4.0.1 are provided

### PR DESCRIPTION
**Pull request by komainu8**
This is testing now.

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -108,8 +108,8 @@ jobs:
           sudo gem install \
             groonga-client \
             json \
-            pkg-config \
-            red-arrow
+            pkg-config
+          sudo gem install red-arrow -v 4.0.0
```
Comment (kou) [inline .github/workflows/linux.yml:112]: We don't need this now.

File: packages/apt/test.sh [hunk 2]
```diff
@@ -54,7 +54,7 @@ gem install grntest
 if groonga --version | grep -q apache-arrow; then
   apt install -V -y \
     g++
-  gem install red-arrow
+  gem install red-arrow -v 4.0.0
```
Comment (kou) [inline packages/apt/test.sh:57]: We don't need this now.

**Comment by HashidaTKS** [comment]
Just asking, are these version specification removed if the packages of Apache Arrow 4.0.1 are provided?

**Comment by kou** [comment]
Yes.
We'll be able to avoid this by improving Apache Arrow's release process. (Uploading Ruby packages after APT, Yum, MSYS2 and Homebrew packages are updated.)

**Review by HashidaTKS — APPROVED** [review]

**Comment by kou** [comment]
Apache Arrow packages are updated. We don't need this.

**Pull request closed without merging**